ansible_collections/arista/avd/roles/eos_designs/docs/tables/bgp-settings.md

```diff
@@ -159,6 +163,18 @@
         password: <str>
         bfd: <bool; default=True>
 
+        # Specify the BFD timers to override the default values.
+        # It is recommended to keep BFD total timeout longer than the DPS timeout.
+        # The Default BFD timeout is 10 x 1 seconds and the default DPS timeout is 5 x 1 seconds.
+        bfd_timers:
+
+          # Interval in milliseconds.
+          interval: <int; 50-60000; default=1000; required>
+
+          # Rate in milliseconds.
+          min_rx: <int; 50-60000; default=1000; required>
+          multiplier: <int; 3-50; default=10; required>
+
```

Comment:
rr_overlay_peers are used for other purposes as well, so we cannot implement these defaults as this would be considered a breaking change. If we need this, we may have to add another `wan_rr_overlay_peers` or just reuse `wan_overlay_peers`.
@gmuloc comments?

Reply:
So the plan is as follow:
1/ For now implement the default timers for BFD peers for RR_OVERLAY_PEERS behind some guard in the code. (So no schema change for `RR_OVERLAY_PEERS`, only for WAN_OVERLAY_PEERS` in this PR)
2/ I will track an issue to separate RR_OVERLAY_PEERS with WAN_RR_PEERS in our tracker as we start to be getting multiple different requirements that could impact functionality

Reply:
@gmuloc Since you already have #3591 ready, I think we should hold back this PR until #3591 merged and then refactor this to set the defaul bfd timers on the wan rr peer-group

Reply:
Have updated now using the wan_rr_overlay_peers.